### PR TITLE
Add log week9

### PR DESCRIPTION
**Pull request by chrarvi**
Woho for audio in our log!

### CODE REVIEW

File: docs/log.org [hunk 1]
```diff
@@ -12,14 +12,99 @@
...
+will continue working on these and later connecting it with our note generation!
+
+*** Training resources
+We got access to another training platform, Bayes at DS&AI division. This server
+has much better hardware than the previous one, but restrictions when it comes
+to time slots to train and amount of training. We'll eventually use it to most
+likely train the transformer since it requires better specs than we had.
+
+*** Meetings
+Meetings has been going well online, we try to work more in voice calls a limit
```
Comment (elias-sundqvist) [inline docs/log.org]: It should be "and limit" here, right?

File: docs/log.org [hunk 2]
```diff
@@ -12,14 +12,99 @@
...
+implementing our own. This week, we managed to run both and generate results.
+Below are the audio snippets of the two.
+
+*Our music transformer with prior*
+
+{{{AUDIO(audio/mutrans_our_no_prior.wav)}}}
+
+*Our music transformer without prior*
+
+{{{AUDIO(audio/mutrans_our_prior.wav)}}}
```
Comment (ericnorman98) [inline docs/log.org]: This one had prior as well

File: docs/log.org (hunk 1 above)
Comment (chrarvi) [reply docs/log.org]: Fixed it

**Review by ericnorman98 — CHANGES_REQUESTED** [review]

File: docs/log.org (hunk 2 above)
Comment (chrarvi) [reply docs/log.org]: So all had priors?

**Review by fantomsol — DISMISSED** [review]
Very good. (There is some grammar stuff that we could poke around in, but nothing important for the whole, so I'm just going to approve. )

**Review by ericnorman98 — APPROVED** [review]

**Pull request merged**